src/flamenco/runtime/fd_blockstore.c

```diff
@@ -548,6 +548,64 @@ fd_blockstore_buffered_shreds_remove( fd_blockstore_t * blockstore, ulong slot )
   return FD_BLOCKSTORE_OK;
 }
 
+ulong
+fd_blockstore_block_archive( fd_blockstore_ser_t * ser, int fd, ulong slot ) {
```

Comment:
generally a public API will always take the `fd_blockstore_t * blockstore` as first arg otherwise if just a helper method can make it a `static` function without the `fd_blockstore` namespacing prefix

(even if unused for uniformity)